src/iologindata.cpp

```diff
@@ -165,6 +165,20 @@ uint32_t IOLoginData::getAccountIdByPlayerName(const std::string& playerName)
 	return result->getNumber<uint32_t>("account_id");
 }
 
+uint32_t IOLoginData::getAccountIdByPlayerId(uint32_t accountId)
```

Comment:
Wrong argument name, it's playerId, not accountId

Reply:
Oops! I hadnt noticed it was edited. I must have broken something with the commit below.